src/aspire/estimation/kernel.py

```diff
@@ -24,6 +24,19 @@ def __init__(self, kernel, centered):
         # TODO: `centered` should be populated based on how the object is constructed, not explicitly
         self._centered = centered
 
+    def __add__(self, regularizer):
+        """
+        There is often a need to add a regularization parameter (a small positive value) to a FourierKernel object,
```

Comment:
Not sure I understand..did you mean to say just make it a generic `add` operation without mentioning regularization (and changing argument names), or not have the operation at all (i.e. allowing the underlying `ndarray` to be modified). The latter scenario won't quite work without refactoring a bunch of things (since the `FourierKernel` object is remembered beyond this optimization procedure and needs to remember its `kernel` value without modifications).

Reply:
> did you mean to say just make it a generic add operation without mentioning regularization (and changing argument names)

Yes.

Reply:
Makes sense..modified the method signature/docstrings to use `delta` instead of `regularizer`. Added a note about why this method even exists, in a `note` docstring directive.